deploy/kubernetes/overlays/prow-gke-release-staging-head/csi-driver.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: storage.k8s.io/v1beta1
+kind: CSIDriver
+metadata:
+  name: pd.csi.storage.gke.io
+spec:
+  attachRequired: true
+  podInfoOnMount: false
```

Comment:
nits:
the file complains "no newline at the end of file"

Reply:
Done